Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent { label 'go'}
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 90, unit: 'MINUTES')
+    }
+    environment {
+        OPENSHIFT_API = "https://api.kogito.automation.rhmw.io:6443"
+        OPENSHIFT_REGISTRY = "default-route-openshift-image-registry.apps.kogito.automation.rhmw.io"
+
+        WORKING_DIR = "/home/jenkins/go/src/github.com/kiegroup/kogito-cloud-operator/"
+
+        MAVEN_MIRROR_REPOSITORY = "http://nexus3-kogito-tools.apps.kogito.automation.rhmw.io/repository/maven-public"
+    }
+    stages {
+        stage('Clean Workspace') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    deleteDir()
+                }
+            }
+        }
+        stage('Clone repository') {
+            steps {
+              sh "mkdir -p ${WORKING_DIR}"
+              sh "git clone https://github.com/kiegroup/kogito-cloud-operator.git ${WORKING_DIR}"
+            }    
+        }
+        stage('Initialize') {
+            steps {
+                sh "set +x && oc login --token=\$(oc whoami -t) --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
+            }
+        }
+        stage('Build Kogito Operator') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh """
+                        export GOPATH=/home/jenkins/go
+                        export GOCACHE=\${GOPATH}/.cache/go-build
+                        export GOROOT=`go env GOROOT`
+                        GO111MODULE=on 
```

Comment:
Is this still needed?
What GO version is used?

Reply:
Right now we are on 1.13. This env was used to use the new go modules back in time, I guess we won't need it anymore since `mod` is default now, but we have to run some tests to make sure.

Reply:
environments variables may be put into `environment` section I think or directly into the running container.
For the gomodule, last time I checked, I needed it to install golint... worth to retry....

Reply:
done!

Reply:
The `export` should not be needed as you set the variables as environment. They will be automatically available

Reply:
ahh okay! I will fix that.

Reply:
done!